### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xliank**
# Overview
README.md contain duplicate like Example code, but there is a example folder. Some title is confusing to read, and i change some description too. Feel free to comment. Thanks

# Test Plan
no needed

### CODE REVIEW

**Comment by MateusAndrade** [comment]
Thanks for your PR @xliank. But I think we dont need to add the sh to the commands on the MD. This can lead to a miss-understanding for some people.

File: README.md [hunk 1]
```diff
@@ -1,41 +1,35 @@
...
-
-### Manual install
+```sh 
+yarn add react-native-share
+react-native link react-native-share
+```
 
-`npm install react-native-share --save`
+We recommend using the releases from npm, however you can use the master branch if you need any feature that is not available on NPM. By doing this you will be able to use unreleased features, but the module may be less stable. 
+**yarn**: ```sh yarn add react-native-share@git+https://git@github.com/react-native-community/react-native-share.git```
```
Comment (MateusAndrade) [inline README.md]: remove sh

File: README.md [hunk 2]
```diff
@@ -1,41 +1,35 @@
...
 
-We recommend using the releases from npm, however you can use the master branch if you need any feature that is not available on NPM. By doing this you will be able to use unreleased features, but the module may be less stable. 
+## Automatic Way
 
-**yarn**: `yarn add react-native-share@git+https://git@github.com/react-native-community/react-native-share.git`
 
-**npm**: `npm install --save react-native-share@git+https://git@github.com/react-native-community/react-native-share.git`
-
-### Manual install
+```sh
```
Comment (MateusAndrade) [inline README.md]: remove sh

File: README.md (hunk 1 above)
Comment (xliank) [reply README.md]: > remove sh

Sure, sh was there to tell the script was need to open on shell or command line. I will update this to comment.

File: README.md [hunk 3]
```diff
@@ -126,8 +140,8 @@ Supported options:
 
 | Name  | Type     | Description |
 | :---- | :------: | :--- |
-| url | string   | URL you want to share (you can share a base64 file url only in iOS & Android ) |
-| urls | Array[string]   | URL's you want to share, Only for iOS and Android (you can share a base64 file url only in iOS & Android ) |
+| url | string   | URL you want to share (only support base64 string in iOS & Android ). |
+| urls | Array[string]   | array of base64 string you want to share (only support iOS & Android). |
```
Comment (kkbhav) [inline README.md]: It supports file URI as well, not only base64 strings. Also, sharing multiple base64 strings is not an efficient way to share files.

Comment (jgcmarins) [reply README.md]: Thanks, @kkbhav.
Guess we need another PR.

**Pull request merged**